homeassistant/strings.json

```
@@ -19,6 +19,7 @@
     "config_flow": {
       "title": {
         "oauth2_pick_implementation": "Pick Authentication Method",
+        "reauth": "Reauthenticate Integration",
```

We should not set uppercase in the middle of sentences right ?

```suggestion
        "reauth": "Reauthenticate integration",
```

Same for `oauth2_pick_implementation` then.

---

It's not really a sentence, though. I think we should follow the APA Style Guide: https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case